application/src/main/java/org/togetherjava/tjbot/config/Config.java

```diff
@@ -426,4 +426,8 @@ public List<RSSFeed> getRssFeeds() {
     public int getRssPollInterval() {
         return rssPollInterval;
     }
+
+    public String getJavaNewsChannelPattern() {
+        return javaNewsChannelPattern;
+    }
```

Comment:
@nateweisz Currently each RSS feed has its own target channel, like here:
```json
"rssFeeds": [
  {
    "url": "<rss_url_feed_here>",
    "targetChannelPattern": "<target_channel_pattern_here>"
  }
]
```
Should we make it so that all RSS feeds point to one text channel?

Reply:
you still can though, its flexible

Reply:
> you still can though, its flexible

Maybe we could have both patterns, with the `targetChannelPattern` as an optional, and the `javaNewsChannelPattern` acting as the fallback pattern to use. What do you think?

Reply:
yeah that sounds like a better solution

Reply:
I added this feature in this commit https://github.com/Together-Java/TJ-Bot/pull/1037/commits/3c6301151b19801203975452af8935dd89335090